src/Parser/Hunter/BeastMastery/Modules/Features/Checklist.js

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+
+import CoreChecklist, { Rule, Requirement, GenericCastEfficiencyRequirement } from 'Parser/Core/Modules/Features/Checklist';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import LegendaryCountChecker from 'Parser/Core/Modules/Items/LegendaryCountChecker';
+import LegendaryUpgradeChecker from 'Parser/Core/Modules/Items/LegendaryUpgradeChecker';
+import PrePotion from 'Parser/Core/Modules/Items/PrePotion';
+import Wrapper from 'common/Wrapper';
+import SPELLS from 'common/SPELLS';
+import CastEfficiency from 'Parser/Core/Modules/CastEfficiency';
+import AlwaysBeCasting from 'Parser/Hunter/BeastMastery/Modules/Features/AlwaysBeCasting';
+import AMurderOfCrows from 'Parser/Hunter/BeastMastery/Modules/Talents/AMurderOfCrows';
+import SpellLink from 'common/SpellLink';
+import Icon from "common/Icon";
+import EnchantChecker from 'Parser/Core/Modules/Items/EnchantChecker';
+import DireBeast from 'Parser/Hunter/BeastMastery/Modules/Spells/DireBeast/DireBeast';
+import BestialWrathAverageFocus from 'Parser/Hunter/BeastMastery/Modules/Spells/BestialWrath/BestialWrathAverageFocus';
+
+class Checklist extends CoreChecklist {
+  static dependencies = {
+    combatants: Combatants,
+
+    //general
+    legendaryUpgradeChecker: LegendaryUpgradeChecker,
+    legendaryCountChecker: LegendaryCountChecker,
+    prePotion: PrePotion,
+
+    //features:
+    castEfficiency: CastEfficiency,
+    alwaysBeCasting: AlwaysBeCasting,
+    enchantChecker: EnchantChecker,
+
+    //talents
+    aMurderOfCrows: AMurderOfCrows,
+
+    //Bestial Wrath
+
+    //Dire Beast
+    direBeast: DireBeast,
+    bestialWrathAverageFocus: BestialWrathAverageFocus,
+
+  };
+
+  rules = [
+    new Rule({
+      name: 'Use core spells as often as possible',
+      description: <Wrapper><a href="https://www.icy-veins.com/wow/beast-mastery-hunter-pve-dps-rotation-cooldowns-abilities" target="_blank" rel="noopener noreferrer">More info.</a></Wrapper>,
+      requirements: () => {
+        const combatant = this.combatants.selected;
+        return [
+          new GenericCastEfficiencyRequirement({
```

Comment:
Does it make sense to have an optional talent as the first item? This may make the rest of the requirements jump a bit. That's not a big deal, I recommend doing this in the order that makes most sense priority-wise.